web/html/src/components/datetime/FromNow.tsx

```diff
@@ -1,23 +1,32 @@
-import * as React from "react";
+import { MomentInput } from "moment";
 
 import { localizedMoment } from "utils";
 
+import { getTimeProps } from "./timeProps";
+
 type Props = {
   value?: string | moment.Moment;
   children?: string;
 };
 
 export const FromNow = (props: Props) => {
-  const rawValue = props.value ?? props.children;
-  if (!rawValue) {
+  const input = props.value ?? props.children;
+  if (!input) {
     return null;
   }
-  const value = localizedMoment(rawValue).tz(localizedMoment.userTimeZone);
-  return <span title={value.toUserString()}>{value.fromNow()}</span>;
+
+  const value = localizedMoment(input).tz(localizedMoment.userTimeZone);
+  const { title, dateTime } = getTimeProps(value);
+  return (
+    <time title={title} dateTime={dateTime}>
+      {value.fromNow()}
+    </time>
+  );
 };
 
 // It returns a simple string instead of a component in the `fromNow` localized format
 // Use case: at the moment the `t()` function (to translate strings) does not support complex components
-export const fromNow = (value) => {
+// TODO: Make this obsolete once https://github.com/SUSE/spacewalk/issues/20449 is implemented
```

Comment:
Just out of curiosity: how is the localization tool going to help with this?

Reply:
Once we have a `t()` function that supports components mixed into translations, we can use the `<FromNow />` component instead of this function since then we get the tooltips etc as well, the `fromNow()` function only generates the text.

Reply:
oh ok, *components mixed into translations* would be really cool.